src/devices/bosch.ts

```diff
@@ -908,14 +917,90 @@ const definitions: Definition[] = [
+                    command: value?.command,
+                    payload: value?.payload,
+                };
+                if (key === 'smoke_alarm_state' || key==='intruder_alarm_state') {
+                    dataToSend.cluster = 'ssIasZone';
+                    dataToSend.payload = {data: value};
+                    dataToSend.command = 128;
+                }
+
+                //
+                // the data of the payload can either be decimal:
+                // 1: this disables the intruder alarm state
+                // 46081: this enables the intruder alert siren
+                // 256: this enables the smoke test alarm siren
+                const endpoint = meta.device.getEndpoint(1);
+                await endpoint.command(
+                    dataToSend.cluster,
+                    dataToSend.command,
+                    dataToSend.payload,
+                    {
+                        ...manufacturerOptions,
+                        ...utils.getOptions(meta.mapped, entity),
+                    },
+                );
+                return {state: {alarm_state: dataToSend.payload.data}};
+            },
+            convertGet: async (entity, key, meta) => {
+                switch (key) {
+                case 'smoke_alarm_state':
+                    await entity.read(
+                        'ssIasZone',
+                        [0xf0],
+                        manufacturerOptions,
+                    );
+                    break;
+                case 'intruder_alarm_state':
+                    await entity.read(
+                        'ssIasZone',
+                        [0xf0],
+                        manufacturerOptions,
+                    );
+                    break;
+                default: // Unknown key
+                    throw new Error(
+                        `Unhandled key toZigbee.rbshoszbeu.convertGet ${key}`,
+                    );
+                }
+            },
+        }],
         meta: {},
         configure: async (device, coordinatorEndpoint, logger) => {
             const endpoint = device.getEndpoint(1);
-            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg', 64684]);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg',
+                'ssIasZone',
+                'ssIasWd',
+                'genBasic',
+                64684]);
             await reporting.batteryPercentageRemaining(endpoint);
+            device.defaultSendRequestWhen = 'immediate';
+            device.save();
+            await endpoint.unbind('genPollCtrl', coordinatorEndpoint);
         },
-        exposes: [e.smoke(), e.battery(), e.battery_low(), e.test()],
+        exposes: [e.smoke(), e.battery(), e.battery_low(), e.test(),
+            e
+                .binary(
+                    'intruder_alarm_state',
+                    ea.ALL,
+                    smokeDetectorAlarmState.intruder_on.toString(),
```

Comment:
I think it's better to just have `ON`/`OFF` here and do the conversion to a number in the toZigbeeConverter

Reply:
@vjonas can you check this comment?

Reply:
Oh yes sorry I overlooked this one. Sure!

Reply:
Should be good now.